.github/workflows/hatch_ci.yml

```diff
@@ -105,14 +121,14 @@ jobs:
             -Dsonar.python.ruff.reportPaths=lint.json
 
       - name: SonarQube Quality Gate check
-        if: always() && github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
+        if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
@artyom-morozov could you confirm if the Sonar Qube Gate is running only on SolaceLabs? That's my understanding here. Was planning on adding whitesource/other checks on SolaceDev. I will reusing the same logic here.

Reply:
Im also thinking it makes sense to hardcode the `org` name here where we want Sonar to run. That's to prevent other forks, or if we only want to limit it to one org.